src/seml/commands/add.py

```diff
@@ -223,8 +294,9 @@ def assemble_slurm_config_dict(experiment_slurm_config: dict):
     # Integrate experiment specific config
     slurm_config = merge_dicts(slurm_config_base, slurm_config)
 
-    slurm_config['sbatch_options'] = remove_prepended_dashes(
-        slurm_config['sbatch_options']
+    slurm_config['sbatch_options'] = cast(
+        SBatchOptions,
+        remove_prepended_dashes(cast(Dict[str, Any], slurm_config['sbatch_options'])),
```

Comment:
well, obviously, the document before or after was a violation of the typeddict; otherwise, the function makes no sense. Does one over the other really make a difference?